src/__page_stories__/index.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
Notice that the directory is weird here ☝️  -- Gatsby is weird about putting stories in the `pages` directory, so you have to put them somewhere else. They suggest making a dir just for the page stories on the guide for Storybook:

https://www.gatsbyjs.org/docs/visual-testing-with-storybook/